release-notes.md

```diff
@@ -3,6 +3,65 @@
+* Full Ubuntu 22.04 for regular and appliance deployment.
+* AMS allows configuring a custom APT mirror for all containers.
+* A new AMS charm configuration `use_network_acl` has been added to allow for experimental use of [LXD network ACLs](https://linuxcontainers.org/lxd/docs//howto/network_acls/) <!-- wokeignore:rule=master --> instead of the AMS node controller to restrict network access of Anbox containers.
+* Included Android security updates for November 2022 (see [Android Security Bulletin - November 2022](https://source.android.com/docs/security/bulletin/2022-11-01) for more information).
+* Updated Android WebView to [107.0.5304.105](https://chromereleases.googleblog.com/2022/11/chrome-for-android-update.html).
+* The used NVIDIA driver series is updated to [515](https://docs.nvidia.com/datacenter/tesla/index.html) which is supported until May 2023.
+* The appliance now deploy Ubuntu 22.04 based LXD containers on new deployments. Existing deployments will keep their existing Ubuntu 20.04 ones.
+* AMD GPU support is now automatically detected by the [`ams-lxd`](https://charmhub.io/ams-lxd) Juju charm and the Anbox Cloud Appliance and the kernel `amdgpu` driver configured for best performance and support.
+* The [AWS g4ad instance types](https://aws.amazon.com/ec2/instance-types/g4/) (equipped with AMD v520 GPUs) are now supported by the [Anbox Cloud Appliance images](https://aws.amazon.com/marketplace/pp/prodview-3lx6xyaapstz4) available through the AWS marketplace.
+* The appliance can now be initialized as any Ubuntu user, not just the `ubuntu` one.
+
+#### Bugs
+
+* AC-1066 Security rules are not applied properly when deploying Anbox Cloud with clustering set up
+* AC-1101 Fingerprint support is marked as enabled
+* AC-1102 Android has `ro.serialno` not set
+* AC-1181 RenderThread is stopped when `/dev/anbox_sync` cannot be opened
+* AC-1197 WebRTC data proxy fails with `4/NOPERMISSION` on stop
+* AC-1204 The agent sets incorrect region value for provided metrics
+* AC-1205 Playout delay set via RTP header causes error/warning on receiver
+* AC-1207 Supervisor does not reconnect after agent was down for longer and terminates Anbox
+* AC-1208 `MESA_GLSL_CACHE_DIR` is deprecated; use `MESA_SHADER_CACHE_DIR` instead
+
+[/Details]
+
+[Details=1.15.2]
+
+# 1.15.2 (October 12 2022)
+
+#### New features & improvements
+
+* Included Android security updates for October 2022 (see [Android Security Bulletin - October 2022](https://source.android.com/security/bulletin/2022-10-01) for more information).
+
+## Known issues
+
+n/a
+
+#### Bugs
+
+* AC-1136 All containers were gone after the appliance snap got refreshed
+* AC-1130 Cannot reconnect to stream session after client with unsupported video codecs tried
+* AC-1087 Can't delete an application when the application bootstrap ends up in an error state
+
+[/Details]
+
```

Comment:
This shouldn't be here?

Reply:
Ah yes, my fault. Will drop in an update.